yunity/karrot-frontend
# Place wall & subscriptions

Closes #986 
Needs backend PR https://github.com/yunity/karrot-backend/pull/662

## What does this PR do?

- add place wall
  - add "place" type to latest messages entries
  - add notification chips to sidenav
- add place subscription toggle
  - show subscribed places first in sidenav (even if not "active")
- add "unsubscribe" option to notification toggle (and remake it completely)
  - cannot unsubscribe from private conversations and thread replies
  - don't show message unread status if user is not subscribed
- filter available pickups by subscribed stores and add a info for the user

Pictures are in https://github.com/yunity/karrot-frontend/issues/986#issuecomment-462466800

To do:
- walls of archived places are hidden right now, maybe need to show them in a special mode
- store wall looks very similar to group wall, how can it be less confusing?
- new users won't be subscribed to any store and don't see pickups on the wall - maybe that's fine? but it's not easy to discover...

## Checklist

- [ ] added a test, or explain why one is not needed/possible...
- [ ] asked someone for a code review
- [ ] joined #karrot-dev channel at https://slackin.yunity.org
- [x] added an entry to CHANGELOG.md (description, pull request link, username(s))
